### PR TITLE
Address #1752

### PR DESCRIPTION
**Pull request by katejeffreys**
@abdonrd - Attempting to address https://github.com/Polymer/docs/issues/1752

### CODE REVIEW

File: app/2.0/start/toolbox/deploy.md [hunk 1]
```diff
@@ -195,3 +195,10 @@ guide](https://www.firebase.com/docs/hosting/quickstart.html).
     The URL to your live site is listed in the output. You can also open
     the site in your default browser by running `firebase open hosting:site`.
 
+#### Optional: Edit your Firebase configuration to enable server push
```
Comment (arthurevans) [inline app/2.0/start/toolbox/deploy.md]: Can we verify whether the default build (or the default build when we launch) includes generating the push manifest? Might be that this should live in a different doc if it's more advanced (i.e., if the generated push manifest might need manual tweaking). Recommend we talk to the tools folks for a recommendation.

**Comment by abdonrd** [comment]
The polymer-build now can generate a push-manifest.json file. Maybe can reference?

**Comment by arthurevans** [comment]
I have concerns about putting this here, in a get-up-and-running tutorial, because it really is pretty advanced stuff. Let's hang on to this and see whether we can slot it into another section?

Ideally, we'd talk about generating the push manifest, editing the push manifest, and setting up push in the same place.

**Comment by katejeffreys** [comment]
Ok - I will think about where to put it. Thanks for the feedback!

**Pull request closed without merging**